### PR TITLE
cloud: update quick start for Serverless Tier cluster creation

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 1]
```diff
@@ -21,23 +21,23 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
+3. To help first-time login users to experience with TiDB Cloud features instantly, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` automatically.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+    - To try out TiDB Cloud features with this default cluster, go to [Step 2. Try AI-powered Chat2Query (beta)](#step-2-try-ai-powered-chat2query-beta).
+    - To try creating a new Serverless Tier by your own, take the following operations:
 
-5. Click **Create**.
-
-    The cluster creation process starts and your Serverless Tier cluster will be created in approximately 30 seconds.
+        1. Click **Create Cluster** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:31]:
```suggestion
        1. Click **Create Cluster**.
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 2]
```diff
@@ -27,19 +27,20 @@ This document walks you through the quickest way to get started with TiDB Cloud.
 
 2. [Log in](https://tidbcloud.com/) to your TiDB Cloud account.
 
-3. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click **Create Cluster**.
+    The [**Clusters**](https://tidbcloud.com/console/clusters) list page is displayed by default.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+3. For the first-time login, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically. You can either use this default cluster for the subsequent steps or create a new Serverless Tier by your own.
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:32]:
```suggestion
3. For the first-time login, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically. You can either use this default cluster for the subsequent steps or create a new Serverless Tier on your own.
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 3]
```diff
@@ -27,19 +27,20 @@ This document walks you through the quickest way to get started with TiDB Cloud.
...
 
-5. Click **Create** to create a Serverless Tier cluster.
+    To create a new Serverless Tier by your own, take the following operations:
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:34]:
```suggestion
    To create a new Serverless Tier on your own, take the following operations:
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 4]
```diff
@@ -21,23 +21,23 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
 
-    The plan selection page is displayed by default.
+    The [**Clusters**](https://tidbcloud.com/console/clusters) list page is displayed by default.
 
-3. On the plan selection page, click **Get Started for Free** in the **Serverless Tier** plan.
+3. For the first-time login, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+    - To try out TiDB Cloud features instantly with this default cluster, go to [Step 2. Try AI-powered Chat2Query (beta)](#step-2-try-ai-powered-chat2query-beta).
+    - To try creating a new Serverless Tier by your own, take the following operations:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:29]:
```suggestion
    - To try creating a new Serverless Tier on your own, take the following operations:
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 5]
```diff
@@ -27,19 +27,20 @@ This document walks you through the quickest way to get started with TiDB Cloud.
...
 2. [Log in](https://tidbcloud.com/) to your TiDB Cloud account.
 
-3. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click **Create Cluster**.
+    The [**Clusters**](https://tidbcloud.com/console/clusters) list page is displayed by default.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+3. For the first-time login, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically. You can either use this default cluster for the subsequent steps or create a new Serverless Tier on your own.
 
-5. Click **Create** to create a Serverless Tier cluster.
+    To create a new Serverless Tier on your own, take the following operations:
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:34]:
```suggestion
    To create a new Serverless Tier cluster on your own, take the following operations:
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 6]
```diff
@@ -27,19 +27,20 @@ This document walks you through the quickest way to get started with TiDB Cloud.
...
 
-    Your TiDB Cloud cluster will be created in approximately 30 seconds.
+    1. Click **Create Cluster**.
+    2. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, select a target region of your cluster, and then click **Create**. Your Serverless Tier cluster will be created in approximately 30 seconds.
 
-6. After your TiDB Cloud cluster is created, click your cluster name to go to the cluster overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
+4. Click the target cluster name to go to its overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:39]:
```suggestion
4. After your cluster is created, click the target cluster name to go to its overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
```
image
The "cluster name" is not clickable during the cluster creation process.

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 7]
```diff
@@ -27,19 +27,20 @@ This document walks you through the quickest way to get started with TiDB Cloud.
 
 2. [Log in](https://tidbcloud.com/) to your TiDB Cloud account.
 
-3. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click **Create Cluster**.
+    The [**Clusters**](https://tidbcloud.com/console/clusters) list page is displayed by default.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+3. For new sign-up users, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically. You can either use this default cluster for the subsequent steps or create a new Serverless Tier on your own.
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:32]:
```suggestion
3. For new sign-up users, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically. You can either use this default cluster for the subsequent steps or create a new Serverless Tier cluster on your own.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 8]
```diff
@@ -21,23 +21,23 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
 
-    The plan selection page is displayed by default.
+    The [**Clusters**](https://tidbcloud.com/console/clusters) list page is displayed by default.
 
-3. On the plan selection page, click **Get Started for Free** in the **Serverless Tier** plan.
+3. For new sign-up users, TiDB Cloud creates a default Serverless Tier cluster `Cluster0` for you automatically.
 
-4. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, and then select the region where you want to create your cluster.
+    - To try out TiDB Cloud features instantly with this default cluster, go to [Step 2. Try AI-powered Chat2Query (beta)](#step-2-try-ai-powered-chat2query-beta).
+    - To try creating a new Serverless Tier on your own, take the following operations:
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:29]:
```suggestion
    - To try creating a new Serverless Tier cluster on your own, take the following operations:
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**